app/views/tag_groups/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_for(@tag_group) do |f| %>
```

Comment:
Don't use instance variables inside of partials, pass it explicitly like: `<%= render 'form', tag_groups: @tag_groups %>` and use it with just `tag_groups` here.